src/events/pointer.js

```diff
@@ -1479,6 +1485,8 @@ function pointer(p5, fn){
 
     if(e.pointerType == 'touch'){
       this._activeTouches.delete(e.pointerId);
+    } else {
+      this._setMouseButton(e);
```

Comment:
It looks like if you have two fingers down, and then raise one, this would cause `mouseButton.left` to become false because the pointer that was lifted would now have no left button. It might be prefereable to, when this happens, keep `mouseButton.left` true if any remaining touch still has a left button down.

So maybe that would mean adding a `buttons` property on each active touch, and when an event happens, we loop through `activeTouches` and set `left = activeTouches.some(touch => touch.buttons.left)`? (I think this would also mean using `activeTouches` for all pointer types, not just touch, to still handle the single-pointer desktop case correctly.

Does that behaviour sound like what we want?

Reply:
I think it makes more sense for mouseButton.left to remain true as long as any touch action is present on the screen. Setting it to false in onpointerup, using something like this, might work well:
``` js
if (this._activeTouches.size === 0) {
  Object.keys(this.mouseButton).forEach((key) => (this.mouseButton[key] = false));
}
```

This approach means we treat all active touches on the screen as left touches, correct? In that case, right and center clicks would only be supported by a mouse if I'm not wrong.